test/gtest/ucp/test_ucp_rma.cc

```diff
@@ -87,8 +97,9 @@ class test_ucp_rma : public test_ucp_memheap {
 private:
     /* Test variants */
     enum {
-        FLUSH_EP,
-        FLUSH_WORKER
+        FLUSH_EP      = 0,
```

Comment:
maybe `FLUSH_EP = UCS_BIT(0)`, `FLUSH_WORKER = UCS_BIT(1)` ...
to have:
```
bool is_ep_flush() {
    return GetParam().variant & FLUSH_EP;
}
```

I think it would be easier when we will extend these flags, we won't need to update `is_ep_flush()`
wdyt?

Reply:
removed FLUSH_WORKER and made FLUSH_EP a flag